CHANGELOG.md

```diff
@@ -19,6 +19,11 @@ Main (unreleased)
 - Some blocks in Flow components have been merged with their parent block to make the block hierarchy smaller:
   - `prometheus.scrape > client > http_client_config` is merged into the `client` block. (@erikbaranowski)
 
+- The `loki.process` component now makes use of River's 'enum' blocks. Instead
+  of defining processing stages in nested a `stage` block containing one
+  element each (eg. a `stage > json` hierarchy), the configuration now supports
+  defining each block as `stage.json` directly. (@tpaschalis)
```

Comment:
I would probably remove the reference to the enum block here and just mention that the various stage blocks have merged with `stage`, so that `stage > json` becomes a single block called `stage.json`.

Reply:
Sorry I merged before seeing your comment. I'll amend with a new PR.

Reply:
It's OK, I can address this in my next PR 👍